Robinhood/Robinhood.py

```diff
@@ -1,37 +1,18 @@
 """Robinhood.py: a collection of utilities for working with Robinhood's Private API """
 
-#Standard libraries
 import logging
 import warnings
 
-from enum import Enum
-
-#External dependencies
 from six.moves.urllib.parse import unquote
 from six.moves.urllib.request import getproxies
 from six.moves import input
 
+from enum import Enum
```

Comment:
Enum is not external in py3.

Py2 is given as a courtesy, but we should assume 3.5+ for style